dali/base/dasds.cpp

```diff
@@ -4091,12 +4088,12 @@ bool translateOldFormat(CServerRemoteTree *parentServerTree, IPropertyTree *pare
 
 void CSDSTransactionServer::processMessage(CMessageBuffer &mb)
 {
-    // 1) Decrement sds_pending_requests here.
+    // 1) Increment sds_requests.started here.
     // NB: request has started, no longer pending
-    // See CSDSTransactionServer::run() for where needs to be incremented. 
+    // See CSDSTransactionServer::run() for where needs to be incremented.
 
-    // 2) Ensure sds_active_requests is decremented, when processMessage() is finished
-    // e.g. use a scoped object here, and decrement in dtor.
+    pSdsRequestsStarted->inc(1);  // this is where the active count was incremented before
```

Comment:
Comment was informational only for the review in case anyone wanted to know what change this represented. I intended on removing it for the final commit. I will remove it now.

Reply:
minor: not removed yet.

Reply:
I actually removed it this time.

Reply:
I don't think any of the remaining comments above useful (and slightly confusing)